docs/guides/HelloGgez.md

```diff
@@ -0,0 +1,70 @@
+# Hello ggez! 👋
+
+We're going to create a very simple program using `ggez`.
+It will say "Hello ggez!" and the current delta time each iteration of the loop.
+
+## Prerequisites ⚠
+
+### Rust
+
+[Rust](https://www.rust-lang.org) will need to be installed.
+Follow Rust's guides for installing the latest stable version.
+
+### SDL
+
+`ggez` requires the [`SDL2` crate](https://github.com/Rust-SDL2/rust-sdl2#user-content-requirements) to be setup on your computer.
+Follow the `SDL2` crate's guide for installing the crate.
+
+## Project Setup 🛠
+
+### Cargo Crate
+
+Open up your terminal and make sure it is in the location where you want to create your project folder.
+
+```bash
+cargo new --bin hello_ggez
+cd hello_ggez
+```
+
+Open up `Cargo.toml` with a text editor and add `ggez` as a dependency:
+
+```toml
+[package]
+name = "hello_ggez"
+version = "0.1.0"
+# Replace with your name and email
+authors = ["Awesome Person awesome@person.com"]
+
+[dependencies]
+ggez = "0.4"
+```
+
+### Check Project Setup ✔
+
+Test to make sure everything is correct by running `cargo run`.
+
+Be patient. This can take a while on the first run. Subsequent builds will be much faster.
+Be sure it says `Hello, world!` at the bottom before continuing.
+
+## ggez
+
+We have successfully compiled `ggez`, but we haven't used it yet.
+What we have currently is the canonical Rust hello world!
+Let's change that and create a hello world for `ggez`. 😊
+
+### Loop
+
+All games have a loop.
+Often the loop is referred to as a game loop.
+
+Game loops are responsible for:
+
+1. Handling events such as keyboard, mouse, closing window, etc.
+1. Updating state such as player position, health, etc.
+1. Drawing shapes, images, etc.
+
+[`EventHandler`](https://docs.rs/ggez/0.4.0/ggez/event/trait.EventHandler.html) is a loop provided by `ggez` to use in our games.
```

Comment:
I think this is better put as, "EventHandler is a way for us to interface with--and implement our own logic on top of--ggez's internal game loop." or something similar.